source/community/release_notes/palm.rst

```diff
@@ -15,14 +15,109 @@ These are the release notes for the Palm release, the 16th community release of
 Breaking Changes
 ****************
 
+RateXBlock unsupported
+======================
+
+The RateXBlock has been deprecated and is now officially unsupported. `FeedbackXBlock 
+<https://github.com/pmitros/FeedbackXBlock>`_ is the recommended replacement.
+
+Stripe API changes
+==================
+
+Ecommerce now supports the new Stripe Payment Intents API and is no longer using the Stripe Charges API. The legacy 
+Stripe Charges ecommerce payment processor with a frontend in the ecommerce application is upgraded to a Stripe 
+Payment Intents payment processor with a frontend in frontend-app-payment. See the discussion post, `Ecommerce: 
+Changes to Stripe payment processor 
+<https://discuss.openedx.org/t/ecommerce-changes-to-stripe-payment-processor/9457>`_ for more details. 
+
+Ecommerce is still officially deprecated and we continue to work on the `Ecommerce Replacement Project 
+<https://openedx.atlassian.net/wiki/spaces/AC/pages/3617849345/Ecommerce+Deprecation+and+Replacement+Project>`_.
+
+
+Developer changes
+=================
+
+The old developer documentation site has been deprecated and removed. Please use `Open edX Developers - Latest 
+Documentation <https://docs.openedx.org/en/latest/developers/index.html>`_  instead.
+
+part of the Tutor API. This change should not affect most developers, who only use the :code:`Actions` and :code:`Filters` 
+classes (notice the plural) from :code:`tutor.hooks`. If it does affect you, there are upgrade instructions in the 
+discussion post, `Simplifying & documenting the Tutor Hooks API 
+<https://discuss.openedx.org/t/simplifying-documenting-the-tutor-hooks-api/9258>`_
+
+Most of the viewing code for PDF certificates was removed in previous releases. In the Palm release, we closed out 
+this work with an ADR (Application Decision Record) on `Leaving PDF Certificate Fields in Certificates Model 
+<https://github.com/openedx/edx-platform/blob/open-release/palm.master/lms/djangoapps/certificates/docs/decisions/008-certificate-model-remnants.rst>`_. 
+
 
 Learner Experiences
 *******************
 
+New Learner Home Page 
+=====================
+
+The new Learner Home has many of the same features of the old dashboard, with some extended functionality and 
+performance enhancements.
+
```

Comment:
As per our discussion, I think learner-home/learner-dasboard MFE as flagged as expiremental, also it has been suggested by @arbrandes to create a tutor plugin for that. That being said I guess you need to a refrence to that plugin to refer here?

Reply:
@ghassanmas, I think so, yes.  If we're mentioning it in the release notes, it would be better if it lived either in the openedx or overhang orgs.

Reply:
If this is experimental, it should be moved to an "Experimental" part of the release notes as we've done in previous releases

Reply:
The plugin is there https://github.com/ghassanmas/tutor-contrib-mfe-dashboard, let me know how/if we need to move it. I have test it with tutor/tutor-mfe palm branch. and it currenlty to my fork until https://github.com/openedx/frontend-app-learner-dashboard/pull/136 is resolved